mod-patron-blocks/src/main/resources/domain/mod-patron-blocks/features/util/initData.feature

```diff
@@ -0,0 +1,165 @@
+Feature: init data for mod-inventory-storage
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
+
+  @Init
+  Scenario: init data
+    * def instance = read('samples/instance-entity.json')
+    * def holdingsRecord = read('samples/holdings-record-entity.json')
+    * def instanceType = read('samples/instance-type-entity.json')
+    * def location = read('samples/location-entity.json')
+    * def institution = read('samples/institution-entity.json')
+    * def campus = read('samples/campus-entity.json')
+    * def library = read('samples/library-entity.json')
+    * def loanTypes = read('samples/loan-type-entity.json')
+    * def circulationRules = read('samples/circulation-rules-entity.json')
+    * def loanPolicy = read('samples/loan-policy-entity.json')
+    * def overdueFinePolicy = read('samples/overdue-fine-policy-entity.json')
+    * def lostItemFeePolicy = read('samples/lost-item-fee-policy-entity.json')
+    * def requestPolicy = read('samples/request-policy-entity.json')
+    * def patronNoticePolicy = read('samples/patron-notice-policy-entity.json')
+    * def servicePoint = read('samples/service-point-entity.json')
+    * instance.instanceTypeId = instanceType.id
+    * instance.instanceTypeId = instanceType.id
+
+    Given path 'service-points'
+    And request servicePoint
+    When method POST
+    Then status 201
+
+    Given path 'loan-policy-storage/loan-policies'
+    And request loanPolicy
+    When method POST
+    Then status 201
+
+    Given path 'overdue-fines-policies'
+    And request overdueFinePolicy
+    When method POST
+    Then status 201
+
+    Given path 'lost-item-fees-policies'
+    And request lostItemFeePolicy
+    When method POST
+    Then status 201
+
+    Given path 'request-policy-storage/request-policies'
+    And request requestPolicy
+    When method POST
+    Then status 201
+
+    Given path 'patron-notice-policy-storage/patron-notice-policies'
+    And request patronNoticePolicy
+    When method POST
+    Then status 201
+
+    Given path 'circulation-rules-storage'
+    And request circulationRules
+    When method PUT
+    Then status 204
+
+    Given path 'loan-types'
+    And request loanTypes
+    When method POST
+    Then status 201
+
+    Given path 'material-types'
+    And request { name: 'book', id: '#(materialTypeId)'}
```

Comment:
Should we etract this request into a file like the others?

Reply:
In other tests, small requests do not have their own file.